setting-up-tugboat/index.md

```diff
@@ -0,0 +1,510 @@
+# Setting up your Tugboat
+
+- [Connect with your provider](#connect-with-your-provider)
+- [Create a new project](#create-a-new-project)
+- [Add repos to the project](#add-repos-to-the-project)
+- [Select repo settings (optional)](#repository-settings-optional)
+- [Create a Tugboat Config file](#create-a-tugboat-config-file)
+
+> #### Hint::Video Overview
+>
+> If you want to learn more about setting up Tugboat and get a feel for the
+> overall workflow, watch our
+> [Getting Started with Tugboat video](https://www.youtube.com/watch?v=HYTsrm5ORmU)
+> on YouTube.
+
+## Connect with your provider
+
+- [Generic git server](#generic-git-server)
+- [GitHub](#github)
+- [GitLab](#gitlab)
+- [BitBucket](#bitbucket)
+- [Add a link to another git provider](#adding-a-link-to-a-git-provider)
+
+### Generic Git Server
```

Comment:
I think this should go to the end of the list, so we highlight our more feature-rich provider integration first, and provide "Generig Git Server" as a fallback if those don't work for you